friends/src/actions/index.js

```diff
@@ -5,6 +5,8 @@ export const FETCHED = "FETCHED";
 export const ERROR = "ERROR";
 export const ADD_FRIEND = "ADD_FRIEND";
 export const FRIEND_ADDED = "FRIEND_ADDED";
+export const DELETE_FRIEND = "DELETE_FRIEND";
+export const FRIEND_DELETED = "FRIEND_DELETED";
 
 export const fetchFriends = () => {
```

Comment:
Hey Shannon. Just in case the format looks the same here as your project, you can make sure your code is clean and indented.